cllc-public-app/ClientApp/src/app/bceid-confirmation/bceid-confirmation.component.html

```diff
@@ -30,7 +30,7 @@ <h2>Confirm Business BCeID Information</h2>
         <input type="radio" name="InitialBusinessType" value="PrivateCorporation" [(ngModel)]="businessType"> <span>Private Corporation</span><br>
         <input type="radio" name="InitialBusinessType" value="PublicCorporation" [(ngModel)]="businessType"> <span>Public Corporation</span><br>
         <input type="radio" name="InitialBusinessType" value="SoleProprietor" [(ngModel)]="businessType"> <span>Sole Proprietorship</span><br>
-        <input type="radio" name="InitialBusinessType" value="GeneralPartnership" [(ngModel)]="businessType"> <span>Partnership</span><br>
+        <input type="radio" name="InitialBusinessType" value="Partnership" [(ngModel)]="businessType"> <span>Partnership</span><br>

```

Comment:
In order for this to work we need a corresponding change to the Enum.   Can you make a corresponding change to the enumeration, setting the value for Partnership to the same as GeneralPartnership for the moment?